assets/src/edit-story/components/panels/textStyle/useRichTextFormatting.js

```diff
@@ -112,19 +114,24 @@ function useRichTextFormatting(selectedElements, pushUpdate) {
         handleClickBold: () => selectionActions.toggleBoldInSelection(),
         // All these keep their arguments:
         handleSelectFontWeight: selectionActions.setFontWeightInSelection,
+        handleResetFontWeight: (weight) => {
+          push(htmlFormatters.setFontWeight, weight);
+        },
         handleClickItalic: selectionActions.toggleItalicInSelection,
         handleClickUnderline: selectionActions.toggleUnderlineInSelection,
         handleSetLetterSpacing: selectionActions.setLetterSpacingInSelection,
         handleSetColor: selectionActions.setColorInSelection,
       };
     }
 
-    const htmlFormatters = getHTMLFormatters();
-
     return {
       handleClickBold: (flag) => push(htmlFormatters.toggleBold, flag),
-      handleSelectFontWeight: (weight) =>
-        push(htmlFormatters.setFontWeight, weight),
+      handleSelectFontWeight: (weight) => {
+        push(htmlFormatters.setFontWeight, weight);
+      },
```

Comment:
Hm, the arrow function is wrapped in a function body to avoid the return value?

Reply:
nope, this was something I changed to just track updates while i was figuring out how everything worked (so i could follow what weight was getting passed in) and forgot to revert. thanks for flagging.